### DISCUSSION TITLE
Add new user id on running stream

### DISCUSSION THREAD

**Discussion by seracoder** [tweepy/tweepy#1847] [Questions] [answered]
Is it possible to add new userid to follow on running stream?

**Comment by Harmon758** [comment, marked as answer]
With Twitter API v2's filtered stream, yes.
https://developer.twitter.com/en/docs/twitter-api/tweets/filtered-stream/introduction
https://tweepy.readthedocs.io/en/v4.7.0/streamingclient.html#tweepy.StreamingClient.add_rules

**Reply by seracoder** [reply]
I mean on Tweepy AsyncStream

**Reply by Harmon758** [reply]
No, that uses Twitter API v1.1.
For async support for Twitter API v2's filtered stream, see #732.
